samples/net/rpl-mesh-qemu/node/CMakeLists.txt

```diff
@@ -5,6 +5,6 @@ set(CONF_FILE prj_qemu.conf)
 set(QEMU_PIPE_STACK 1)
 
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
-project(NONE)
+project(node)
```

Comment:
Better and more descriptive would be `rpl_mesh_qemu_node`

Reply:
I can see the pattern you are suggesting. Will update accordingly, likely for all projects, not just the ones you suggested to improve.